src/styles/index.scss

```diff
@@ -1,3 +1,21 @@
+$card-color: #fff;
+$border-color: #f3f3f3;
+$font-main-color: #060b35;
+$font-secondary-color: #616070;
+$inactive-button-color: #00acdc;
+$active-button-color: #fff;
+$inactive-button-font-color: #fff;
+$active-button-font-color: #00acdc;
+$active-button-border-color: #00acdc;
```

Comment:
there is no sense to create two variables with different names for one variable
The sense of variables is to have a value in one place so that you can change it in one place and it will change everywhere

What if you used #fff 100 times through your project,  would you create 100 different names for it? or just one and use it everywhere?